jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/HttpChannel.java

```diff
@@ -1530,7 +1530,7 @@ public void failed(final Throwable x)
             if (x instanceof HttpException httpException)
             {
                 MetaData.Response responseMeta = new MetaData.Response(httpException.getCode(), httpException.getReason(), HttpVersion.HTTP_1_1, HttpFields.build().add(HttpFields.CONNECTION_CLOSE), 0);
-                send(_request.getMetaData(), responseMeta, null, true, new Nested(this)
+                send(_request.getMetaData(), responseMeta, null, true, new Nested(this.getCallback())
```

Comment:
No need for `this.getCallback()`, just `getCallback()`.

Reply:
It is needed as I have removed the strange "copy" constructor.    The previous behaviour relied on the fact that the passed callback was a nested, so that instead of being wrapped, it was replaced.  This was entirely non intuitive and relied on the type of the passed callback, which a user might not even know that the callback they have is actually implemented as a nested.... so if they strongly type it, they get different behaviour.  Specifically the following code:
```java
SomeCallback callback = new SomeCallback();
call(new Nested(callback));
```
may not have the same behaviour as:
```java
Callback callback = new SomeCallback();
call(new Nested(callback));
```

This is fragile (cost me many hours trying to figure out), so I removed the "copy" constructor and the replacement of the nested callback is now explicit in the error handling here.

Reply:
Sure, I meant to remove `this.` and just have `new Nested(getCallback())`.